### PR TITLE
JBPM-5457: Start and Stop container buttons change states even though they fail

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/JBPM-5457

Part of an ensemble:
- https://github.com/kiegroup/droolsjbpm-integration/pull/1066
- https://github.com/kiegroup/kie-wb-common/pull/999

### CODE REVIEW

**Comment by karreiro** [comment]
Jenkins retest this

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/ContainerPresenter.java [hunk 1]
```diff
@@ -139,184 +95,239 @@ public ContainerPresenter( final Logger logger,
...
+                content.getContainers() != null) {
+            setup(content.getContainerSpec(),
+                  content.getContainers());
         } else {
-            logger.warn( "Illegal event argument." );
+            logger.warn("Illegal event argument.");
         }
     }
 
+    public void refreshOnContainerUpdateEvent(@Observes final ContainerUpdateEvent updateEvent) {
```
Comment (karreiro) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/main/java/org/kie/workbench/common/screens/server/management/client/container/ContainerPresenter.java]: The real change is here :-)

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/test/java/org/kie/workbench/common/screens/server/management/client/container/ContainerPresenterTest.java [hunk 2]
```diff
@@ -121,208 +122,281 @@
...
-        presenter.load( new ContainerSpecSelected( containerSpec ) );
+        presenter.load(new ContainerSpecSelected(containerSpec));
 
-        verifyLoad( true, 1 );
+        verifyLoad(true,
+                   1);
+    }
+
+    @Test
+    public void testRefreshOnContainerUpdateEvent() {
```
Comment (karreiro) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-client/src/test/java/org/kie/workbench/common/screens/server/management/client/container/ContainerPresenterTest.java]: ...and here.

**Comment by mswiderski** [comment]
this can't be merged until https://github.com/kiegroup/droolsjbpm-integration/pull/1066 has resolved conflicts

**Pull request merged**